src/App.js

```diff
@@ -5,6 +5,7 @@ import 'bootstrap/dist/css/bootstrap.min.css';
 import './fonts/GoogleSans-Bold-v1.27.ttf';
 import './fonts/GoogleSans-Medium-v1.27.ttf';
 import './fonts/GoogleSans-Regular-v1.27.ttf';
+import 'bootstrap/dist/js/bootstrap.js';
```

Comment:
NO NO NO, we DON'T WANT THIS !

Reply:
Sir in case the bootstrap js is not required then the navbar won't be responsive. The bootstrap package comes together with both the CSS and JS bundles. At the level of the drop-down on mobile on medium devices it can't work without.

Reply:
Alternatively, we'll have to create or mange that effect on our own, which will make the work pretty much slow. As suggestion this PR should be merged while I'm implementing this feature to the drop-down. I'm saying so to avoid some merge conflicts and mismatch of some values, some of the changes I made are to be reused to continue with the other tasks.